src/components/ebay-combobox/index.js

```diff
@@ -63,6 +73,18 @@ module.exports = require('marko-widgets').defineComponent({
                 simulateSpacebarClick: true
             });
         }
+
+        if (isExpanded && !wasExpanded) {
```

Comment:
Wont this code not do anything?

This assumes that isExpanded is true and wasExpanded is false. That means that wasToggled will also be true (since isExpanded !== wasExpanded) which will execute line 81.

Reply:
It will do something, but you're right that it is basically duplicated in the following if block(s).